auth0/resource_auth0_user_test.go

```diff
@@ -0,0 +1,74 @@
+package auth0
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccUserMissingRequiredParams(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]terraform.ResourceProvider{
+			"auth0": Provider(),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config:      testAccUserMissingRequiredParam,
+				ExpectError: regexp.MustCompile(`config is invalid: auth0_user.user: "conn": required field is not set`),
+			},
+		},
+	})
+}
+
+const testAccUserMissingRequiredParam = `
+provider "auth0" {}
+
+resource "auth0_user" "user" {}
+`
+
+func TestAccUserCreateUser(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]terraform.ResourceProvider{
+			"auth0": Provider(),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccUserCreateUser, os.Getenv("AUTH0_CLIENT_ID")),
```

Comment:
Could also create a client for the purposes of this test. See client grants tests or resource server.